### PR TITLE
Create user story template

### PR DESCRIPTION
**Pull request by cunninghamjc**
Created a custom issue template for user stories.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/user-story.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: User story
```
Comment (jackliums) [inline .github/ISSUE_TEMPLATE/user-story.md]: No bug template?

Comment (cunninghamjc) [reply .github/ISSUE_TEMPLATE/user-story.md]: I already submitted that: #205

File: .github/ISSUE_TEMPLATE/user-story.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+about: Tell us something new you want to do
+labels:
```
Comment (jackliums) [inline .github/ISSUE_TEMPLATE/user-story.md]: Do we want user to enter type labels here?

Comment (cunninghamjc) [reply .github/ISSUE_TEMPLATE/user-story.md]: No, the user doesn't actually see or edit this part; it seems to be metadata used by github on the "new issue" page.  I haven't tried adding labels to the template to see what it does, but I would guess that it adds these labels automatically to the issue.  I don't think we'd do it for user stories, but we could try it on bugs.

**Pull request merged**